org.previewqt.PreviewQt.yml

```diff
@@ -0,0 +1,306 @@
+    ########################
+    - name: ghostscript
+      config-opts:
+        - --enable-shared
+        - --disable-static
+      sources:
+        - type: archive
+          url: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10021/ghostscript-10.02.1.tar.gz
+          sha256: e429e4f5b01615a4f0f93a4128e8a1a4d932dff983b1774174c79c0630717ad9
+    ########################
+    - name: imagemagick
+      builddir: true
+      build-options:
+      config-opts:
+        - --enable-shared
+        - --disable-static
+        - --disable-docs
+        - --with-modules
+        - --with-threads
+        - --with-png
+        - --enable-hdri
+        - --enable-opencl
+        - --with-djvu
+        - --with-fftw
+        - --with-jxl
+        - --with-lqr
+        - --with-modules
+        - --with-openexr
+        - --with-openjp2
+        - --with-rsvg
+        - --with-webp
+        - --with-wmf
+        - --with-xml
+        - --with-dps
+        - --with-fpx
+        - --with-gvc
+      sources:
+        - type: git
+          url: https://github.com/ImageMagick/ImageMagick
+          tag: 7.1.1-29 # when changing the version number make sure to update patch file accordingly
+          commit: 0deac72ed480ac2ec8e9d766c15ddb3bca055952
+
+    ########################
+    ########################
+    - name: previewqt
+      buildsystem: cmake-ninja
+      builddir: true
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DIMAGEMAGICK=ON
+        - -DGRAPHICSMAGICK=OFF
+        - -DEXIV2=ON
+        - -DLIBARCHIVE=ON
+        - -DRAW=ON
+        - -DPOPPLER=ON
+        - -DQTPDF=OFF
+        - -DDEVIL=ON
+        - -DFREEIMAGE=OFF
+        - -DLIBVIPS=OFF
+        - -DVIDEO_MPV=OFF
+        - -DEXIV2_ENABLE_BMFF=ON
+      post-install:
+        - mv /app/share/icons/hicolor/16x16/apps/org.photoqt.PreviewQt.png /app/share/icons/hicolor/16x16/apps/org.previewqt.PreviewQt.png
+        - mv /app/share/icons/hicolor/32x32/apps/org.photoqt.PreviewQt.png /app/share/icons/hicolor/32x32/apps/org.previewqt.PreviewQt.png
+        - mv /app/share/icons/hicolor/48x48/apps/org.photoqt.PreviewQt.png /app/share/icons/hicolor/48x48/apps/org.previewqt.PreviewQt.png
+        - mv /app/share/icons/hicolor/64x64/apps/org.photoqt.PreviewQt.png /app/share/icons/hicolor/64x64/apps/org.previewqt.PreviewQt.png
+        - mv /app/share/icons/hicolor/128x128/apps/org.photoqt.PreviewQt.png /app/share/icons/hicolor/128x128/apps/org.previewqt.PreviewQt.png
+        - mv /app/share/icons/hicolor/256x256/apps/org.photoqt.PreviewQt.png /app/share/icons/hicolor/256x256/apps/org.previewqt.PreviewQt.png
+        - mv /app/share/icons/hicolor/512x512/apps/org.photoqt.PreviewQt.png /app/share/icons/hicolor/512x512/apps/org.previewqt.PreviewQt.png
+        - rm -r /app/share/icons/hicolor/1024x1024/
+        - mv /app/share/applications/org.photoqt.PreviewQt.desktop /app/share/applications/org.previewqt.PreviewQt.desktop
```

Comment:
`rename-desktop-file` exist for this use case.

Reply:
I released v2.0 of this app today that fixes this upstream.